docssrc/src/chatpreview.md

```diff
@@ -22,13 +22,13 @@ For players that want to use chat preview, they must have `Chat Preview` enabled
 
 ## Specifying a chat preview function
 
-The `ChatArgument` and `AdventureChatArgument` classes include a method, `withPreview`:
+The `ChatArgument` classes include a method, `withPreview`:
 
 ```java
 public T withPreview(PreviewableFunction preview);
 ```
 
-The method `withPreview(PreviewableFunction preview)` lets you generate a preview to send to the client. This method takes in the `PreviewableFunction` functional interface, which is a function that takes in a `PreviewInfo` and returns either a `BaseComponent[]` (for `ChatArgument`) or a `Component` (for `AdventureChatArgument`):
+The method `withPreview(PreviewableFunction preview)` lets you generate a preview to send to the client. This method takes in the `PreviewableFunction` functional interface, which is a function that takes in a `PreviewInfo` and returns either a `BaseComponent[]` (for the Spigot-based `ChatArgument`) or a `Component` (for the Adventure-based`ChatArgument`):
```

Comment:
Just a tiny typo, it looks like there's a space missing. 
```suggestion
The method `withPreview(PreviewableFunction preview)` lets you generate a preview to send to the client. This method takes in the `PreviewableFunction` functional interface, which is a function that takes in a `PreviewInfo` and returns either a `BaseComponent[]` (for the Spigot-based `ChatArgument`) or a `Component` (for the Adventure-based `ChatArgument`):
```
<img width="747" alt="Screen Shot 2023-09-25 at 10 10 27 AM" src="https://github.com/JorelAli/CommandAPI/assets/46540330/a7d0f8ab-d0ff-46f1-b76f-1423a3d63bc4">